08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="css/homework-style.css" type="text/css" />
+    <title>Lesson 8 In Class Project</title>
+</head>
+
+<body>
+
+
+  <div id="page-wrap">
+
+
+    <header>
+        <h1>Recent From Blog</h1>
+        <center>
+
+
+        <p1>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla accumsan suscipit lacinia. </br>
+              Vivamus euismod varius odio eget cursus. Nulla accumsan suscipit lacinia.
+            </p1>
+                     </center>
+    </header>
+    </br>
+    </br>
+
+    <section id="keyboard">
+        <img id="keyboard_image" src="img/keyboard.JPG" alt="keyboard">
+        <h3>Lorem ipsum dolor sit amet.</h3>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla accumsan suscipit lacinia. Vivamus euismod varius odio eget.
+        </p>
+        <button type="button">Read More </button>
+
+    </section>
+
+    <aside>
+
+        <section class="product">
+            <img id="laptop" src="img/laptop.JPG" alt="laptop">
```

Comment:
this image and the next one have .jpg extensions. the first one was the only image with the capitalized extension. just go off whichever the exact file uses in the ATOM file tree